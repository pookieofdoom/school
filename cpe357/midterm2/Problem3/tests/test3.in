5   54 -39 -78 26 82 
3   22 90 -15
5
