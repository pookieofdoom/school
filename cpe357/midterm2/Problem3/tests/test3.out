54, -39, -78, 26, 82, 22, 90, -15
Expected amount of memory used
